src/qcodes/instrument_drivers/Keithley/Keithley_2400.py

```diff
@@ -134,6 +147,7 @@ def __init__(
             "Note that it is an error to read current "
             "and voltage with output off",
         )
+        """Measure resistance from current and voltage Note that it is an error to read current and voltage with output off"""
```

Comment:
```suggestion
        """Measure resistance from current and voltage. Note that it is an error to read current and voltage with output off"""
```

Reply:
Do this manually to also resolve the docstr argument